src/components/Pagination/index.js

```diff
@@ -68,12 +68,12 @@ class Pagination extends React.Component {
   lastPageLink( pager ) {
     return pager.isLastPage() ?
       <li className="disabled">
-        <a href="#" aria-label="Next">
+        <a href="#" aria-label="Next" onClick={ function(e){ e.preventDefault(); } }>
           <span aria-hidden="true">{ this.props.labels.last }</span>
         </a>
       </li> :
       <li onClick={ this.setPage.bind( this, pager.total - 1 ) }>
-        <a href="#" aria-label="Next">
+        <a href="#" aria-label="Next" onClick={ function(e){ e.preventDefault(); } }>
           <span aria-hidden="true">{ this.props.labels.last }</span>
         </a>
       </li>;
```

Comment:
We're all on the same page then :)

On Wed, May 20, 2015, 11:53 AM Tim Carry <notifications@github.com> wrote:

> In src/components/Pagination/index.js
> <https://github.com/algolia/algolia-ui-kit/pull/11#discussion_r30685923>:
>
> >            <span aria-hidden="true">{ this.props.labels.last }</span>
> >          </a>
> >        </li> :
> >        <li onClick={ this.setPage.bind( this, pager.total - 1 ) }>
> > -        <a href="#" aria-label="Next">
> > +        <a href="#" aria-label="Next" onClick={ function(e){ e.preventDefault(); } }>
> >            <span aria-hidden="true">{ this.props.labels.last }</span>
> >          </a>
> >        </li>;
>
> I'd rather see a real link (with real href) too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/algolia/algolia-ui-kit/pull/11/files#r30685923>.
>